auth.css

```
@@ -118,3 +120,24 @@ h3 {
 .signup-go a {
     color: #3182f6;
 }
+
+/* Mobile: 375px~767px */
+@media (max-width: 767px) {
```

💊 제안
주석을 다는 것은 좋은 습관입니다. 다만 주석또한 읽어야하는 정보라 중복되지 않는 내용을 입력해주시는 것이 가독성면에서 더 좋아요.
지금 주석은 아래 미디어쿼리 조건을 통해 파악할 수 있는 내용이고 실제로는 max-width만 존재하기 때문에 없거나 아래정도로 수정하시는 것이 좋을 것 같아요~

```suggestion
/* Mobile */
@media (max-width: 767px) {
```